src/event.cpp

```diff
@@ -128,13 +128,13 @@ wcstring event_get_desc(const event_t &e) {
             if (e.param1.pid > 0) {
                 result = format_string(_(L"exit handler for process %d"), e.param1.pid);
             } else {
-                job_t *j = job_get_from_pid(-e.param1.pid);
+                job_t *j = job_get_from_pid(e.param1.pid);
```

Comment:
This change was wrong; the negative PID is used to store a PGID (yowsers). I've backed this out and added a comment, although it's ripe for rearchitecting.